wiki/Beatmap/Genre_and_language/de.md

```diff
@@ -0,0 +1,67 @@
+---
+tags:
+  - genres
+  - languages
+  - metadata
+  - Sprachen
+---
+
+# Genre und Sprache
+
+![Ein Screenshot des Genres, der Sprache und die Tags auf einer Beatmap-Seite](img/beatmap-info-DE.png "Genre und Sprache werden auf Beatmap-Seiten angezeigt.")
+
+[Beatmaps](/wiki/Beatmap), die auf die Webseite hochgeladen werden, haben Felder für das **Genre** und die **Sprache**, um bei der Kategorisierung von Songs zu helfen. Sie werden als Teil der [Metadaten](/wiki/Client/Beatmap_editor/Song_setup#song-and-map-metadata) einer Beatmap angesehen.
+
+Das Genre und die Sprache haben keine Funktion im Spiel, sind aber als Suchoptionen in [osu!(lazer)](/wiki/Client/Release_stream/Lazer) verfügbar.
+
+## Beatmap-Auflistung
+
+![Die verfügbaren Filter in der Beatmap-Auflistung](img/beatmap-listing-DE.png)
+
+Die [Beatmap-Auflistung](https://osu.ppy.sh/beatmapsets) hat Filter für das Genre und die Sprache im Dropdown-Menü `Mehr Suchoptionen`.
+
+## Das Genre und die Sprache einer Beatmap ändern
+
+Das Genre und die Sprache kann auf der Webseite geändert werden, indem man mit dem Mauszeiger über die Details einer Beatmap fährt und dabei auf das Bleistiftsymbol klickt. Dabei bestimmen die [Benutzergruppe](/wiki/People/User_group) des Nutzers und die [Kategorie](/wiki/Beatmap/Category#aktuelle-kategorien) der Beatmap die jeweiligen Rechte:
+
+- Beatmap-Eigentümer dürfen das, wenn sich die Beatmap in der Kategorie [WIP](/wiki/Beatmap/Category#work-in-progress-und-ausstehend), [Ausstehend](/wiki/Beatmap/Category#work-in-progress-und-ausstehend) oder [Friedhof](/wiki/Beatmap/Category#friedhof) befindet und keine [Nominierungen](/wiki/Beatmap_ranking_procedure#nominations) hat.
```

Comment:
`dürfen das, wenn...` in this section doesn't read great. I see that the structure of the english article makes it a bit difficult to translate...

Maybe `können Änderungen vornehmen, wenn...`? It's more verbose though.

Reply:
Yeah, good point. I second guessed this part too. I will take another look at it.

Reply:
1f5018b9e776a78fd7ab7078cd78b992615b46f4